db-test/src/androidTest/java/com/islamversity/db_test/InsertQuranTextTest.kt

```diff
@@ -0,0 +1,484 @@
+package com.islamversity.db_test
+
+import android.util.Log
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.google.common.truth.Truth.assertThat
+import com.islamversity.db.Aya
+import com.islamversity.db.Main
+import com.islamversity.db.Sora
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlDriver
+import kotlinx.coroutines.async
+import kotlinx.coroutines.test.runBlockingTest
+import kotlinx.serialization.json.Json
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.File
+import kotlin.system.measureTimeMillis
+
+/*
```

Comment:
Why has been commented?

Reply:
needs to be changed.
due to the DB schema changes